src/services/navigation/useNavLinks/useNavLinks.js

```diff
@@ -376,14 +383,15 @@ export function useNavLinks() {
     },
     componentsTab: {
       path: (
-        { provider = p, owner = o, repo = r, branch = undefined } = {
+        { provider = p, owner = o, repo = r, branch = b } = {
           provider: p,
           owner: o,
           repo: r,
+          branch: b,
         }
       ) => {
         if (branch) {
-          return `/${provider}/${owner}/${repo}/components?branch=${branch}`
+          return `/${provider}/${owner}/${repo}/components/${branch}`
```

Comment:
What do you think about doing like `/${provider}/${owner}/${repo}/components/${encodeURIComponent(branch)}` so that we don't have to remember to call encode at all usages?

Reply:
this makes sense! I'll add this change